projects/schematics/src/shared/utils/method-call-utils.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { findNodes } from '@schematics/angular/utility/ast-utils';
+import * as ts from 'typescript';
+import { parseTsFileContent } from './file-utils';
+
+interface ReplaceMethodCallArgumentParams {
+  fileContent: string;
+  objectName: string;
+  methodName: string;
+  argument: {
+    position: number;
+    oldText: string;
+    newText: string;
+  };
+  throwErrorIfNotFound?: boolean;
+}
+
+interface RemoveMethodCallParams {
+  fileContent: string;
+  objectName: string;
+  methodName: string;
+  throwErrorIfNotFound?: boolean;
+}
+
+interface FindMethodCallsParams {
+  fileContent: string;
+  objectName: string;
+  methodName: string;
+}
+
+/**
+ * Finds all method calls in the given file content.
+ *
+ * Returns the method calls.
+ */
+export function findMethodCalls({
+  fileContent,
+  objectName,
+  methodName,
+}: FindMethodCallsParams): ts.CallExpression[] {
+  const sourceFile = parseTsFileContent(fileContent);
+
+  // Get all method calls
+  const nodes = findNodes(
+    sourceFile,
+    ts.SyntaxKind.CallExpression,
+    undefined,
+    true
+  );
+
+  return nodes.filter((node): node is ts.CallExpression => {
+    if (!ts.isCallExpression(node)) {
+      return false;
+    }
+    const expression = node.expression;
+    if (!ts.isPropertyAccessExpression(expression)) {
+      return false;
+    }
+
+    const object = expression.expression;
+    const method = expression.name;
+
+    if (!ts.isIdentifier(object) || !ts.isIdentifier(method)) {
+      return false;
+    }
+
+    return object.text === objectName && method.text === methodName;
+  });
+}
+
+export function removeMethodCalls({
```

Comment:
`minor:` if I'm not mistaken, this function is not used anywhere. Let's remove it.